src/api.js

```diff
@@ -0,0 +1,19 @@
+const API_END_POINT = 'http://0.0.0.0:3000/';
+
+export const API = {
+  UPLOAD_RESUME: '/resume/upload',
+};
+

```

Comment:
```suggestion
const BASE_URL = 'http://0.0.0.0:3000';

export const api = {
  upload_resume: `${BASE_URL}/resume/upload`
};

// api.upload_resume
```
이렇게 하면,BASE_URL을 가져다 쓰지않아도  api 객체 하나만으로도 전역에서 사용하기 편하겠네요